public/js/user/home.js

```diff
@@ -23,6 +23,12 @@ formContactUs.addEventListener('submit', (e) => {
   formData.forEach((value, key) => {
     dataContactUs[key] = value;
   });
+  const { email, fullName, message } = dataContactUs;
+  if (!email || /^[\w.-_%+]+@[\w.-]+\.[a-zA-Z]{2,4}$/.test(email) === false || fullName || message) {
```

Comment:
@aminalakhsham I think you need ` || !fullname || !message)`, since you're looking for empty inputs.

Reply:
what the hake
it remove by mistake

Reply:
haha 🤷‍♂️